Dockerfile

```diff
@@ -11,15 +11,15 @@ RUN apk --no-cache add \
 	make
 
 ENV COREDNS_VERSION v1.6.9
-ENV DNSSEEDER_VERSION v0.2.1
+ENV DNSSEEDER_VERSION v0.2.2
 
 RUN git clone --depth 1 --branch ${COREDNS_VERSION} https://github.com/coredns/coredns /go/src/github.com/coredns/coredns
 
 WORKDIR /go/src/github.com/coredns/coredns
 
 RUN echo "dnsseed:github.com/zcashfoundation/dnsseeder/dnsseed" >> /go/src/github.com/coredns/coredns/plugin.cfg
 # Must be the same replace as in `dnsseeder`. Currently pointing to "main-zfnd" branch
-RUN echo "replace github.com/btcsuite/btcd => github.com/ZcashFoundation/btcd v0.22.0-beta.0.20211118133831-ca5d3008dd64" >> /go/src/github.com/coredns/coredns/go.mod
+RUN echo "replace github.com/btcsuite/btcd => github.com/ZcashFoundation/btcd v0.22.0-beta.0.20220607000607-40dc9492aa42" >> /go/src/github.com/coredns/coredns/go.mod
```

Comment:
Check if this is the same replace that is in `go.mod` in `dnsseeder` at the version pointed above

Reply:
✅